crypto/init.c

```diff
@@ -73,7 +73,9 @@ struct ossl_init_stop_st {
 };
 
 static CRYPTO_RWLOCK *glock_lock = NULL;
+#ifdef OPENSSL_SYS_UNIX
 static CRYPTO_ONCE glock_once = CRYPTO_ONCE_STATIC_INIT;
```

Comment:
Actually, I would simply move this down to the function where it's used...  perhaps even inside it.

Reply:
Good point, done.